setup.cfg

```diff
@@ -0,0 +1,21 @@
+[flake8]
+exclude =
+    .git,
```

Comment:
Are all these ignores really necessary? If we're specifying the module/directory, surely it should only check that directory, and know not to lint the `__pycache__` dir?

Reply:
flake8's docs include the suggestion of including the `__pycache__` and `.git` directories.
I mainly included them as I'm not sure what arguments editors will call flake8 with.